src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SkippableFactAttribute.cs

```diff
@@ -34,6 +35,12 @@ public static void CheckEnvs(bool skipOnNullOrWhiteSpace, bool skipOnTrue, Actio
                 setSkip($"Skipping because `{envName}` is set to True");
                 break;
             }
+            // Skip OmniSharp tests for ppc64le and s390x
```

Comment:
The scope of this change applies to any test that uses `SkippableFact` though, not just OmniSharp.

A couple solutions I can think of:
* Create a derived class of `SkippableFactAttribute` that implements this behavior and have OmniSharp test use that attribute.
* Update this class to take a list of architectures that should be skipped (ideally this would be optional for callers). Then have OmniSharp pass in these architectures.

Reply:
I like the suggestion of supporting an architecture list to skip on because it feels more broadly applicable in the future for other scenarios.